### ISSUE TITLE
Error creating new Wiki pages on group projects in Gitlab 8.7.0

### ISSUE THREAD

**Issue by kmetsalu** [gitlabhq/gitlabhq#10317]
Im getting this error on group projects. In user own namespace I can create wiki pages. 

What should I check?

```bash
NoMethodError (undefined method `username' for #<Group:0x00000008ff57a0>):
  app/models/project_wiki.rb:159:in `default_message'
  app/models/project_wiki.rb:152:in `commit_details'
  app/models/project_wiki.rb:91:in `create_page'
  app/models/wiki_page.rb:183:in `save'
  app/models/wiki_page.rb:143:in `create'
  app/services/wiki_pages/create_service.rb:6:in `execute'
  app/controllers/projects/wikis_controller.rb:59:in `create'
  lib/gitlab/middleware/go.rb:16:in `call'
```

**Comment by rymai** [comment]
@kmetsalu Thanks for the report, this will be fixed by https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/3924.

**Comment by MrChrisW** [comment]
Closing as https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/3924 is merged and will be included in v8.7.2
